Scoop/Controller/ProfileViewController.swift

```diff
@@ -50,7 +60,11 @@ class ProfileViewController: UIViewController, ProfileViewDelegate {
         setupEditButton()
         setupProfileImageView()
         setupProfileStackView()
-        updateProfile()
+        if !isBeingPresented {
+            updateUserProfile()
+        } else {
+            updateDriverProfile()
+        }
```

Comment:
You could use a ternary operator here, but I don't know what the standard is. If not then I would also do `if isBeingPresented` first and swap the two function calls. Just feels a lot more natural for slow people like me.